proctor-codegen/src/main/resources/com/indeed/proctor/consumer/ant/groups.ftl

```diff
@@ -184,6 +187,27 @@ public class ${mainClassName} extends AbstractGroups {
         return payload.${testDef.payloadAccessorName}();
     }
     </#if>
+
+    <#if (testDef.isJson)??>
+    public @Nullable <T> T get${testDef.javaClassName}Payload(final Class<T> payloadType) {
+        final Payload payload = getPayload(${testEnumName}.${testDef.enumName}.getName(), ${testEnumName}.${testDef.enumName}.getFallbackValue());
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
nitpick: It may be better to define it as a `static` variable so that we can reuse the same instance for each method call.